app/screens/client_upgrade/client_upgrade.js

```diff
@@ -235,7 +235,7 @@ export default class ClientUpgrade extends PureComponent {
                         version: currentVersion,
                     }}
                 />
-                {isUpgradeAvailable(this.state.upgradeType) &&
+                {isUpgradeAvailable(upgradeType) &&
```

Comment:
is this the only change required for the upgrade screen? Will this change enable the link to download new apks from the update notification screen

Reply:
Yes,this change shows the "Update" and "Update Later" option in the upgrade screen.The downloading functionality is already implemented .